ern-container-gen/src/utils.js

```diff
@@ -423,8 +423,8 @@ export function injectReactNativeVersionKeysInObject (
   reactNativeVersion: string) {
   return Object.assign(object, {
     reactNativeVersion,
-    RN_VERSION_GTE_49: semver.gte(reactNativeVersion, '0.49.0'),
```

Comment:
hey @deepueg PR looks good to me. please confirm `RN_VERSION_GTE_49` is not used elsewhere in the repo.